openapi/openapi.yaml

```diff
@@ -641,6 +641,8 @@ components:
           deleted_at:
             type: string
             format: date-time
+          meta_data:
```

Comment:
maybe `meta` or `metadata`

Reply:
I think we should use `meta_data` here, to keep the format consistent with other fields, e.g.

```
    consumer_name:
            type: string
...
          delete_option:
            type: object
          manifest_configs:
          ```

Reply:
treat  `meta_data` as a whole world